tests/search/HomeSearch.test.tsx

```diff
@@ -0,0 +1,123 @@
+import HomeSearch from '@src/components/inputs/Search/HomeSearch';
+import { decodeDomainToASCII, lowerCaseDomain } from '@src/utils';
+import { queryClient } from '@src/utils/network';
+import { QueryClientProvider } from '@tanstack/react-query';
+import { cleanup, render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { act } from 'react';
+
+jest.mock('@src/hooks/useArNSRegistryDomains', () => ({
+  useArNSRegistryDomains: jest.fn().mockReturnValue({
+    data: {
+      items: [
+        {
+          processId: 'I-cxQhfh0Zb9UqQNizC9PiLC41KpUeA9hjiVV02rQRw',
+          startTimestamp: 1711122719,
+          type: 'permabuy',
+          undernameLimit: 10,
+          purchasePrice: 0,
+          name: 'ardrive',
+        },
+        {
+          processId: 'I-cxQhfh0Zb9UqQNizC9PiLC41KpUeA9hjiVV02rQRw',
+          startTimestamp: 1711122719,
+          type: 'lease',
+          undernameLimit: 10,
+          purchasePrice: 0,
+          name: 'xn--go8h6v',
+        },
+      ],
+    },
+    isLoading: false,
+  }),
+}));
+
+describe('HomeSearch', () => {
```

Comment:
before we invest more time on these formatted test, if they are not on the ideal testing pattern path we want to get to (e.g. playwright) then let's focus on migrating to those tools and adding tests after.

Reply:
What we have here is a good format for unit tests I think that we could use in conjunction with playwright, but honestly I have used playwright much so it may indeed be redundant.